utilities/virt.py

```diff
@@ -1369,25 +1368,33 @@ def vm_console_run_commands(
         vm (obj): VirtualMachine
         commands (list): List of commands
         timeout (int): Time to wait for the command output
-        verify_commands_output (bool): Check commands return 0
-        command_output (bool): If selected, returns a dict of command and associated output
+        return_code_validation (bool): Check commands return 0
+
+    Returns:
+        Dict of the commands outputs, where the key is the command and the value is the output as a list of lines.
     """
     output = {}
     # Source: https://www.tutorialspoint.com/how-can-i-remove-the-ansi-escape-sequences-from-a-string-in-python
     ansi_escape = re.compile(r"(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]")
-    with Console(vm=vm) as vmc:
+    with Console(vm=vm, prompt=r"\$ ") as vmc:
         for command in commands:
             LOGGER.info(f"Execute {command} on {vm.name}")
-            vmc.sendline(command)
-            vmc.expect(r".*\$")
-            output[command] = ansi_escape.sub("", vmc.after).replace("\r", "").split("\n")
-            if verify_commands_output:
-                vmc.sendline("echo rc==$?==")  # This construction rc==$?== is unique. Return code validation
-                try:
+            try:
+                vmc.sendline(command)
+                vmc.expect(vmc.prompt)
```

Comment:
This will not work because `vmc` is not the `Console` instance, but the spawn child in it.
Same for the other usages.

Reply:
Done